commonmark-ext-gfm-tables/src/main/java/org/commonmark/ext/gfm/tables/TableCell.java

```diff
@@ -32,6 +33,17 @@ public void setAlignment(Alignment alignment) {
         this.alignment = alignment;
     }
 
+    /**
+     * @return the cell width
```

Comment:
Can you add a description of what this mean here, similar to the PR description?

Reply:
I've done that now..